lib/bot.js

```diff
@@ -36,6 +36,11 @@ function launchBot (team, cb) {
   if (!team.bot) { return cb(new Error('team has no bot: ' + team)) }
   const bot = slack.rtm.client()
   const token = team.bot.bot_access_token
+  bot.team_join( data => {
+    require('./events/team_join')(token, data, e => {
+      console.log('sent welcome message')
+    })
+  })
```

Comment:
Oh that event wasn't working and I wasn't sure if it was deprecated or something or inactive. Thanks for activating I will change that.